cvat/apps/engine/static/engine/js/base.js

```diff
@@ -128,11 +128,83 @@ function showOverlay(message) {
 }
 
 
-async function dumpAnnotationRequest(dumpButton, tid) {
-    dumpButton = $(dumpButton);
-    dumpButton.attr('disabled', true);
+async function showInput(messageValue, defaultValue = '') {
+    return new Promise((resolve) => {
+        const template = $('#inputTemplate');
+        const inputWindow = $(template.html()).css('display', 'block');
+        const message = inputWindow.find('.templateMessage');
+        const input = inputWindow.find('.templateInputText');
+        const agreeConfirm = inputWindow.find('.templateAgreeButton');
+        const disagreeConfirm = inputWindow.find('.templateDisagreeButton');
+
+        message.text(messageValue);
+        input.prop('value', defaultValue);
+
+        function hideInput() {
+            agreeConfirm.off('click');
+            disagreeConfirm.off('click');
+            input.off('keydown');
+            inputWindow.remove();
+        }
+
+        agreeConfirm.on('click', () => {
+            hideInput();
+            resolve(input.prop('value'));
+        });
+
+        disagreeConfirm.on('click', () => {
+            hideInput();
+            resolve(null);
+        });
+
+        input.on('keydown', (e) => {
+            e.stopPropagation();
+        });
+
+        $('body').append(inputWindow);
+    });
+}
 
-    // TODO
+
+function validateDumpName(dumpName) {
+    const restrictPattern = /[/-\w]+/g;
+    return !dumpName.replace(restrictPattern, '');
+}
+
+async function dumpAnnotationRequest(tid) {
+    const name = await showInput('Please enter a file name');
```

Comment:
@bsekachev ,

I really don't like the idea to ask the user the filename. It is annoying. The server sends the filename in its response. Just use it.